content/en/security/infrastructure_vulnerability/_index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Infrastructure Vulnerability Management
+kind: documentation
+is_beta: true
+further_reading:
+ - link: "/security/infrastructure_vulnerability/setup"
+   tag: "Documentation"
+   text: "Setting up Infrastructure Vulnerability Management"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">
+Infrastructure Vulnerability Management is not available on the US1-FED Datadog site.
+</div>
+{{< /site-region >}}
+
+{{< callout url="https://forms.gle/GwAWNEMSqyoao2fX6" >}}
+  Infrastructure Vulnerability Management is in private beta, but you can easily request access! Use this form to submit your request today.
+{{< /callout >}}
+
+Datadog [Infrastructure Vulnerability Management][1] uses the Datadog Agent to monitor your container images and identify known vulnerabilities for you to prioritize and remediate. The Agent scans running container images on hosts and creates a software bill of materials (SBOM). The SBOM is then scanned for vulnerabilities.
```

Comment:
Datadog [Infrastructure Vulnerability Management][1] leverages deep infrastructure observability to detect, prioritize, manage and remediate vulnerabilities in your organization. It allows you to track the vulnerabilities to closure with the understanding of your infrastructure, and its ownership, across your organization.

This capability is currently in Private Beta. It offers vulnerabilities in the Container Images on your hosts. If you are enrolled in the Private Beta, you will see additional capabilities light up with existing configuration.